modules/twinklefluff.js

```diff
@@ -566,15 +566,11 @@ Twinkle.fluff.callbacks = {
 Twinkle.fluff.formatSummary = function(builtInString, userName, userString) {
 	var result = builtInString;
 
-	// append user's custom reason with requisite punctuation
+	// append user's custom reason
 	if (userString) {
 		result += ': ' + Morebits.string.toUpperCaseFirstChar(userString);
-		if (userString.search(/[.?!;]$/) === -1) {
-			result += '.';
-		}
-	} else {
-		result += '.';
 	}
+	
```

Comment:
Extra tab.

```suggestion

```

Reply:
Fixed. I left it for adequate resolving space probably.